help/en/releasenotes/current-draft-note.shtml

```diff
@@ -85,6 +85,11 @@
                 <li>Added a DecoderPro definition for the DS78V. This required
                     code changes, so this definition will not work with earlier
                     versions of JMRI.</li>
+                <li>The jython/loconet/LnEditEvolutionRoutesAndOpSw.py script has been updated
+                    to version 2.0.
+                    This script allows you to configure the OpSw and routes in the new Digitrax
```

Comment:
Does Digitrax use the term "Digitrax               Series 7 Evolution boards"?  I've heard of "Evolution" to refer to DCS210/DCS210+ command stations, but not other "modern" command stations (DCS240/DCS240+/DCS52) or "modern" LocoNet accessories.

Perhaps it would be appropriate to talk about "7th-generation" LocoNet accessories and the "Evolution" (DCS210/210+), "Advanced"(DCS240/240+), and "Zephyr Express" (DCS52) command stations.

Reply:
I think you're right about that.  I can's find a reference to "evolution" referring to the boards this can program.  Will update.